sdk/keyvault/azure-security-keyvault-certificates/inc/azure/keyvault/certificates/certificate_client_operations.hpp

```diff
@@ -23,16 +23,17 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Certificat
    * @brief Represents a create certificate long running operation
    */
   class CreateCertificateOperation final
-      : public Azure::Core::Operation<CertificateOperationProperties> {
+      : public Azure::Core::Operation<KeyVaultCertificateWithPolicy> {
 
     friend class CertificateClient;
 
   private:
     std::shared_ptr<CertificateClient> m_certificateClient;
-    CertificateOperationProperties m_value;
+    KeyVaultCertificateWithPolicy m_value;
+    CertificateOperationProperties m_properties;
```

Comment:
I think this can be public?  Or what's the reason to keep it private?

Reply:
so it's read only , i don't want poeole to modify it, especially while polling